### DISCUSSION TITLE
Can I ask you to add Unix badge?

### DISCUSSION THREAD

**Discussion by dann-romm** [Ileriayo/markdown-badges#372] [Ideas]
The request is motivated by considerations of knowledge of Unix processes
this can be added to OS section

**Comment by kohasummons** [comment]
Hey @dan-romm! Yes, you can. We love badge requests. Someone would see this
and contribute it. Also, if you want to go that adventure yourself later
on, you could check out our contribution guide to see how to add to our
repo subsequently.

**Comment by Jukelyn** [comment]
Badge requests should go in the issues page.
